packages/appstore/src/components/containers/trading-app-card.tsx

```diff
@@ -17,140 +16,150 @@ type TWalletsProps = {
+            mt5_acc_auth_status,
+            openFailedVerificationModal,
+            selected_mt5_jurisdiction
+        );
 
-    const openStaticPage = () => {
-        if (platform === CFD_PLATFORMS.MT5 && availability === 'EU')
-            window.open(getStaticUrl(`/dmt5`, {}, false, true));
-        else if (platform === CFD_PLATFORMS.MT5 && availability !== 'EU') window.open(getStaticUrl(`/dmt5`));
-        else if (platform === CFD_PLATFORMS.DXTRADE) window.open(getStaticUrl(`/derivx`));
-        else if (platform === CFD_PLATFORMS.DERIVEZ) window.open(getStaticUrl(`/derivez`));
-        else if (icon === 'Options' && !is_eu_user) window.open(getStaticUrl(`/trade-types/options/`));
-        else;
-    };
+        const openStaticPage = () => {
+            if (platform === CFD_PLATFORMS.MT5 && availability === 'EU')
+                window.open(getStaticUrl(`/dmt5`, {}, false, true));
+            else if (platform === CFD_PLATFORMS.MT5 && availability !== 'EU') window.open(getStaticUrl(`/dmt5`));
+            else if (platform === CFD_PLATFORMS.DXTRADE) window.open(getStaticUrl(`/derivx`));
+            else if (platform === CFD_PLATFORMS.DERIVEZ) window.open(getStaticUrl(`/derivez`));
+            else if (icon === 'Options' && !is_eu_user) window.open(getStaticUrl(`/trade-types/options/`));
+            else;
+        };
 
-    return (
-        <div className='trading-app-card' key={`trading-app-card__${current_language}`}>
-            <div
-                className={classNames('trading-app-card__icon--container', {
-                    'trading-app-card__icon--container__clickable': clickable_icon,
-                })}
-            >
-                <TradigPlatformIconProps icon={icon} onClick={clickable_icon ? openStaticPage : undefined} size={48} />
-            </div>
-            <div className={classNames('trading-app-card__container', { 'trading-app-card--divider': has_divider })}>
-                <div className='trading-app-card__details'>
-                    <div>
+        return (
+            <div className='trading-app-card' key={`trading-app-card__${current_language}`}>
+                <div
+                    className={classNames('trading-app-card__icon--container', {
+                        'trading-app-card__icon--container__clickable': clickable_icon,
+                    })}
+                >
+                    <TradigPlatformIconProps
+                        icon={icon}
+                        onClick={clickable_icon ? openStaticPage : undefined}
+                        size={48}
+                    />
+                </div>
+                <div
+                    className={classNames('trading-app-card__container', { 'trading-app-card--divider': has_divider })}
+                >
+                    <div className='trading-app-card__details'>
+                        <div>
+                            {is_wallet ? (
```

Comment:
@thisyahlen-deriv The component is the same, better to combine them together.

```suggestion
               <Text className='title' size='xs' line_height='s' color='prominent'>
                                    {((is_wallet && is_wallet_demo) || !is_real) && sub_title ? `${sub_title} ${localize('Demo')}` : sub_title}
                                </Text>             
```